src/filters/FreeTextFilter.js

```diff
@@ -11,6 +11,12 @@ export class FreeTextFilter extends Filter {
       let termFound = false
       const fieldNames = Object.keys(record)
       fieldNames.forEach((fieldName) => {
+        if (typeof record[fieldName] !== 'object') { return }
```

Comment:
Do all these checks have equivalent tests?

Reply:
not quite; the final test should fail if any of them fail. Hopefully this code will be rethought next version anyway.